trlx/orchestrator/ppo_orchestrator.py

```diff
@@ -92,7 +114,7 @@ def make_experience(self, num_rollouts: int = 1024, iter_count: int = 0):  # noq
                 )
                 for output in outputs
             ]
-            response_tensors = torch.vstack(outputs).to(device)
+            padded_samples = torch.vstack(outputs).to(device)
```

Comment:
Probably `padded_outputs` (or rather `output_tensors` in view of `query_tensors`) is better suited name here, since by a sample `prompt + output` is implied everywhere else

Reply:
I think `padded_outputs` makes sense - made the change

Reply:
How about `sample_outputs` to be even clearer?